cli/cmd/encore/app/create.go

```diff
@@ -47,6 +51,7 @@ var createAppCmd = &cobra.Command{
 
 func init() {
 	appCmd.AddCommand(createAppCmd)
+	createAppCmd.Flags().BoolVar(&createAppOnPlatform, "platform", true, "whether to create the app with the Encore Platform")
```

Comment:
What does createAppOnPlatform mean? What's the use case of setting this to false?

Reply:
It means the app is only created locally, mainly for local testing purposes